docs/options.md

```diff
@@ -452,14 +442,14 @@ Default: `auto`
 | Windows / ARM64 | `ARM64` | `ARM64` | `ARM64` | |
 | macOS / Intel | `x86_64` | `x86_64` | `x86_64` |  |
 | macOS / Apple Silicon | `arm64` | `arm64` | `arm64` |  |
-
-If not listed above, `auto` is the same as `native`.
```

Comment:
This should be kept below the table ?

Reply:
I'm not sure what the purpose of this sentence is, because it actually is listed above for all supported platforms.

Reply:
Linux s390x/ppc64le/... for example are not listed in the table.

Reply:
Understood - I'll restore the line.